smalltalksrc/VMMaker/StackInterpreter.class.st

```diff
@@ -2984,19 +2990,14 @@ StackInterpreter >> bytecodePrimGreaterThanSistaV1 [
 { #category : #'common selector sends' }
 StackInterpreter >> bytecodePrimIdenticalSistaV1 [
 
-	<compilationInfo: #isMapped>
-	<druidInfo: #hasSend>
 	| rcvr arg |
-	self druidIgnore: [
-		rcvr := self stackValue: 1.
-		arg := self stackValue: 0.
-		(objectMemory isOopForwarded: rcvr) ifTrue: [
-			rcvr := self handleSpecialSelectorSendFaultFor: rcvr ].
-		(objectMemory isOopForwarded: arg) ifTrue: [
-			arg := self handleSpecialSelectorSendFaultFor: arg ].
-		^ self booleanCheatSistaV1: rcvr = arg ].
-
-  self normalSendSpecialSelector: 22 argumentCount: 1
+	rcvr := self stackValue: 1.
+	arg := self stackValue: 0.
+	(objectMemory isOopForwarded: rcvr) ifTrue: [
+		rcvr := self handleSpecialSelectorSendFaultFor: rcvr ].
+	(objectMemory isOopForwarded: arg) ifTrue: [
+		arg := self handleSpecialSelectorSendFaultFor: arg ].
+	^ self booleanCheatSistaV1: rcvr = arg
```

Comment:
So these bytecodes never send the message?

Reply:
It never did in stock VMs. Only in druid.